eventdao.py

```diff
@@ -0,0 +1,14 @@
+from dao import DAO
+
+class EventDAO(DAO):
```

Comment:
Not a bad style to do this, but if we make any changes to the DAO it will affect this Event DAO .. if more than one class extend from the DAO we will need to make sure the DAO is base class structure .. if only one class if ever extending from it, than there is no need for this .. smart thinking though 👍